### ISSUE TITLE
VOCA evaluation

### ISSUE THREAD

**Issue by hdjang** [X-niper/UniTalker#19]
Hi

I evaluated provided pretrained model (UniTalker-B-[D0-D7]) on VOCA test set, and got LVE as below. But its number is higher than one from paper. What should I do to get correct evaluation number?
FYI, I tested both test data identity and random id(934) for evalution, but they showed the same evaluation numbers as below. 

paper: 8.14E-06
my test: 1.06E-05

**Comment by X-niper** [comment]
Hi, do use use the correct identity in evaluation?

**Comment by hdjang** [comment]
Hi !
What is the correct identity?

I tested both 1) test data identity and 2) random id (idx=934). But both numbers show same number of 1.06E-05 as above.

**Comment by X-niper** [comment]
> Hi ! What is the correct identity?
> 
> I tested both 1) test data identity and 2) random id (idx=934). But both numbers show same number of 1.06E-05 as above.

Hi, in our training, BIWI is D0 and its ID is [0,1,..., 5], VOCASET is D1 and its ID range is [6,7,8,..., 17]. For detailed information, please refer to the codes here https://github.com/X-niper/UniTalker/blob/main/dataset/dataset.py#L72 and the train, val, test split meta from Unitalker-data-release-V1.zip

**Comment by hdjang** [comment]
Aha

You mean I should pass the VOCASET "test data ids" among [6,7,8,..., 17], not a random id(943)?

**Comment by X-niper** [comment]
> Aha
> 
> You mean I should pass the VOCASET "test data ids" among [6,7,8,..., 17], not a random id(943)?

Yes, the evaluation protocal is the same as faceformer and codetalker, where the ground truth ID label is used.

**Comment by hdjang** [comment]
Aha !

Thansk for the detailed reply 😆

Following your comments, I finally got the same evaluation numbers as paper on VOCA test set !

Thanks :)

**Comment by X-niper** [comment]
> Aha !
> 
> Thansk for the detailed reply 😆
> 
> Following your comments, I finally got the same evaluation numbers as paper on VOCA test set !
> 
> Thanks :)

Cool. Thank you for your interest in our work
